kapitan/resources.py

```diff
@@ -99,6 +102,19 @@ def jinja2_render_file(search_paths, name, ctx):
 
     raise IOError("jinja2 failed to render, could not find file: {}".format(_full_path))
 
+def yaml_to_json(search_paths, name):
+    """return content of file in json"""
+    for path in search_paths:
```

Comment:
We probably don't need to search multiple paths. Just one file path would be fine for this function in jsonnet.

Reply:
For testing it, I have to create a sample yaml file. Where should I create it?

Reply:
search_paths is necessary to keep it consistent with the other functions